packages/resource-scheduler/src/modules/resourceScheduler/reducer.js

```diff
@@ -0,0 +1,22 @@
+import * as actions from './actions'
+import {reducers} from 'tocco-util'
+
+const addCalendars = (state, {payload}) => ({
+  ...state,
+  calendars: [...state.calendars, ...payload.calendars]
```

Comment:
Maybe also make sure duplicates cannot exist?

Reply:
i d like to solve this as soon as we integrate the selectable change